### ISSUE TITLE
Request: Image preview maximize to fit window

### ISSUE THREAD

**Issue by swieder227** [zanllp/sd-webui-infinite-image-browsing#375]
TL;DR: feature request to add button to image so that it "fits" to the viewport.

When you click on an image, it displays at natural size, and then you have the ability to zoom in and out. Often I want to zoom in on 512x768 images. However, often a single zoom-in will result in the image being cropped since it overflows the viewport.

It would be great if there was a button, up by the zoom, that set the image preview size to the full width/height of the container while maintaining aspect ratio. And that this setting would persist as you scroll through the gallery of pictures.

FWIW, this can be achieved via CSS:
```
img.ant-image-preview-img {
    width: 100%;
    height: 100%;
    object-fit: contain;
}
```

I was digging through the code and unable exactly to find where the image preview element and toolbar are instantiated. It seems they're provided by the ant UI lib.

**Comment by swieder227** [comment]
@zanllp if you can point to me to where that image preview / toolbar is instantiated, I can look into customizing it by adding a new button to the toolbar, persisting an `image-fit: enabled` state, and applying a CSS class to the image preview > `img` when true.

**Comment by zanllp** [comment]
https://github.com/zanllp/sd-webui-infinite-image-browsing/blob/main/vue/src/page/fileTransfer/fullScreenContextMenu.vue

https://github.com/zanllp/sd-webui-infinite-image-browsing/blob/d015f3e86e9474362b4be17a80fd436e5c63b4fd/vue/src/page/fileTransfer/stackView.vue#L220

Recommend using checkboxes or switches to implement this functionality, and persistently save this setting
